lib/server/services/dataPasses/DataPassService.js

```diff
@@ -26,6 +26,14 @@ const { BadParameterError } = require('../../errors/BadParameterError');
  * @param {string} [name]
  */
 
+/**
+ * Lists of default GAQ detectors for runs with given pdp_beam_type
+ */
+const DEFAULT_GAQ_DETECTORS_FOR_PDP_BEAM_TYPE = {
+    pp: ['TPC', 'ITS', 'FT0'],
+    PbPb: ['TPC', 'ITS', 'FT0', 'ZDC'],
+};
+
```

Comment:
I think that a constant should be an array at max, not something complex like that in general.
In this case, I think that here you should simply make 2 constants and create the object where it's needed

Reply:
applied